awxkit/awxkit/api/pages/execution_environments.py

```diff
@@ -40,7 +40,7 @@ def create_payload(self, name='', organization=Organization, credential=None, **
     def payload(self, name='', image=None, organization=None, credential=None, pull='', **kwargs):
         payload = PseudoNamespace(
             name=name or "EE - {}".format(random_title()),
-            image=image or random_title(10),
+            image=image or "example.com/component:tagname",
```

Comment:
```
$ podman pull example.com/component:tagname
Trying to pull example.com/component:tagname...
Error: Error initializing source docker://example.com/component:tagname: error pinging docker registry example.com: invalid status code from registry 404 (Not Found)
```

There are some societal reasons I don't love this, what about this instead?

```
$ podman pull example.invalid/component:tagname
Trying to pull example.invalid/component:tagname...
Error: Error initializing source docker://example.invalid/component:tagname: error pinging docker registry example.invalid: Get "https://example.invalid/v2/": dial tcp: lookup example.invalid: no such host
```

Reply:
fixed